common/rbnf/de_CH.xml

```diff
@@ -168,7 +168,7 @@ x.x: << Komma >>;
 1: ­=%spellout-ordinal=;
 %%ste2:
 0: ste;
-1: ‘ =%spellout-ordinal=;
```

Comment:
The replacement of curly by straight apostrophe is suspicious. I don't think that should be in the transform, or otherwise be special-cased.

Reply:
no, it's converted properly from:

- https://github.com/unicode-org/cldr/blob/6a57fe5413950e6ed03042de179856c47074a28d/common/rbnf/de.xml#L178

so if de.xml has the wrong content, that's another bug (which I'll file).  Maybe @grhoten hand-corrected this line previously?

Reply:
In any case, this line change seems correct to me.